server/src/main/java/com/coincoinche/events/Event.java

```diff
@@ -0,0 +1,15 @@
+package com.coincoinche.events;
+
+public class Event {
```

Comment:
I replaced the SocketMessage class by event class because they are the same and we will reuse events for other purposes.